server/src/pages/homepage/model.js

```diff
@@ -4,6 +4,7 @@ exports.createModel = function(req) {
   const model = {
     title: "Firefox Screenshots",
     showMyShots: !!req.deviceId,
+    hasFxa: !!req.accountId,
```

Comment:
After successful /api/fxa-oauth/login redirect to homepage , req.accountId stays null. Loading shot  or shot index page sets req.accoutId and displays Setting links on homepage. @6a68 @chenba do you know what else needs to be ported into homepage model that's handled in shot pages to reflect req.accountId

Reply:
Not sure what's going on from just reading the code.  The auth middleware should be executing for `/`.